packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js

```diff
@@ -17,13 +17,23 @@ import {
 import { getTrad } from '../../utils';
 import { cleanData, generateOptions } from './utils';
 
-const CMEditViewCopyLocale = ({ appLocales, currentLocale, localizations, readPermissions }) => {
+const CMEditViewCopyLocale = props => {
+  if (!props.localizations.length) {
+    return null;
+  }
```

Comment:
I don't understand why we need to export that condition in another component. 🤔

Reply:
To make sure we always that the options array contains at least one localisation. See my other comment